test/image_test.py

```diff
@@ -1072,6 +1072,74 @@ def get_hash(img: Image) -> str:
     assert get_hash(img) == "bfce5811c04c1243f12cbb9cca1522cb901f52410986925bcfa3b3c2d7adc7a0"
 
 
+@pytest.fixture
+def force_2024_10(modal_config):
+    with mock.patch("test.conftest.ImageBuilderVersion", Literal["2024.10"]):
```

Comment:
Patching a fixture!? 🤯 
For config changes I usually use `monkeypatch.set_env("MODAL_IMAGE_BUILDER_VERSION")` or similar, but not sure if that's possible here.